.github/PULL_REQUEST_TEMPLATE/release.md

```
@@ -0,0 +1,35 @@
+## 📌 작업 요약
+- 요약:
+  - develop 브랜치의 누적 변경사항을 main으로 릴리즈 배포
+- 관련 이슈: closes #
+
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)